src/Components/Ignitor/src/ElementNode.cs

```diff
@@ -100,11 +101,11 @@ public Task ClickAsync(HubConnection connection)
                 Type = clickEventDescriptor.EventName,
                 Detail = 1
             };
-            var webEventDescriptor = new
+            var webEventDescriptor = new WebEventDescriptor
             {
                 BrowserRendererId = 0,
                 EventHandlerId = clickEventDescriptor.EventId,
-                EventArgsType = "mouse",
+                EventName = "mouse",
```

Comment:
click?

Reply:
Well spotted - fixed :)

The test was still passing though.